docs/content/_navigation.json

```diff
@@ -71,6 +71,7 @@
     "title": "Concepts",
     "icon": "Menu",
     "path": "/concepts",
+    "isDefaultOpen": true,
```

Comment:
Having both of these open by default is a bit much - combo of too long and a bit cluttered, due to list items in the **Guides** section having lengthy names. Maybe we just go with **Concepts** for the time being? We can always revisit as part of the consolidation work we're doing.

Reply:
sure - will update. we'll change the nav in the consolidation work anyways and we can keep "the default open section" as a bullet point to decide in nav revamp :)